packages/cspell-types/src/CSpellSettingsDef.ts

```diff
@@ -266,6 +266,10 @@ export interface CommandLineSettings {
      * Define cache settings.
      */
     cache?: CacheSettings;
+    /**
+     * Exit with non-zero code as soon as an issue/error encountered (useful for CI or git hooks); false by default
```

Comment:
In this case, add `@default false`, that way it makes it into the schema.

Reply:
Done. Should I also update docs?

Reply:
> Done. Should I also update docs?

To update `docs/...` it needs to happen after the merged because links in the docs reference the commit sha.